### DISCUSSION TITLE
VS Code code server

### DISCUSSION THREAD

**Discussion by AIIIAN** [coder/code-server#4662] [Q&A]
Hello I have problem with set up code/server can someone please help me.
Here is photo https://linksharing.samsungcloud.com/mJB530QOKYgM
I think that i very close, but when i write ./code-server turmax send me not fond. 
I will be very happy for each advice.

**Comment by lg-nineoneninethreenine** [comment]
Have you installed NodeJS 14 on Termux?

**Reply by AIIIAN** [reply]
No. I am completly new in this world so I installed everythink  with help on ytb, but I have lots of problems. Can someone send me full tutorial how install VS code on android. For example which you tried. 
Thanks for any help.

**Reply by lg-nineoneninethreenine** [reply]
First, you should find how to install nodejs 14 via tarballs on Termux. Next, after installing Node 14 and npm, run `npm i -g code-server@3.10.2` since the latest ones have a bug(#4422)
